nexus/types/src/quiesce.rs

```diff
@@ -151,26 +157,77 @@ impl SagaQuiesceHandle {
         SagaQuiesceHandle { log, inner }
     }
 
-    /// Disallow new sagas from being started or re-assigned to this Nexus
+    /// Set the intended quiescing state
+    ///
+    /// Quiescing is currently a one-way trip.  Once we start quiescing, we
+    /// cannot then re-enable sagas.
+    pub fn set_quiescing(&self, quiescing: bool) {
+        self.inner.send_if_modified(|q| {
+            let new_state = if quiescing {
```

Comment:
Nit - this is _only_ used in the `SagasAllowed::DisallowedUnknown` arm below - can we move it into that arm?